src/nupic/algorithms/SDRClassifier.hpp

```diff
@@ -33,16 +33,15 @@
 #include <string>
 #include <vector>
 
-#include <nupic/algorithms/ClassifierResult.hpp>
+#include <nupic/types/ClassifierResult.hpp>
 #include <nupic/types/Types.hpp>
 #include <nupic/types/Serializable.hpp>
 
 namespace nupic {
 namespace algorithms {
-
-typedef cla_classifier::ClassifierResult ClassifierResult;
-
 namespace sdr_classifier {
+using namespace std;
```

Comment:
Nitpick: `using namespace std` in header?  Is this really needed here?  Usually this line is only in the source files...

Reply:
true. I didn't add that one, just moved it from global scope to within the ::nupic::algorithms::sdr_classifier